packages/ubuntu_wsl_setup/lib/pages/profile_setup/profile_setup_widgets.dart

```diff
@@ -1,5 +1,35 @@
 part of 'profile_setup_page.dart';
 
+class _RealNameFormField extends StatelessWidget {
```

Comment:
You don’t reuse that widget in ubuntu_desktop_installer?

Reply:
Right, the two were developed in parallel by different persons and there were issues with sharing translations. I'll add this on my TODO-list. Thanks! :+1:

Reply:
ack, make sense!